github/search.go

```diff
@@ -71,6 +71,34 @@ func (s *SearchService) Repositories(ctx context.Context, query string, opt *Sea
 	return result, resp, err
 }
 
+// TopicsSearchResult represents the result of a topics search.
+type TopicsSearchResult struct {
+	Total             *int           `json:"total_count,omitempty"`
+	IncompleteResults *bool          `json:"incomplete_results,omitempty"`
+	Topics            []*TopicResult `json:"items,omitempty"`
+}
+
+type TopicResult struct {
+	Name             *string  `json:"name,omitempty"`
+	DisplayName      *string  `json:"display_name,omitempty"`
+	ShortDescription *string  `json:"short_description,omitempty"`
+	Description      *string  `json:"description,omitempty"`
+	CreatedBy        *string  `json:"created_by,omitempty"`
+	CreatedAt        *string  `json:"created_at,omitempty"`
```

Comment:
Please change `*string` to `*Timestamp` here and on the line below as well.

Reply:
done

Reply:
# github.com/google/go-github/v28/github
../../github/github-accessors.go:11924:9: cannot use *t.CreatedAt (type Timestamp) as type string in return argument
../../github/github-accessors.go:11972:10: cannot use t.Score (type *float64) as type *float32 in return argument

Reply:
You need to re-run `go generate ./...`.  Please see `CONTRIBUTING.md` for details.